src/codecs/optipng/options.tsx

```diff
@@ -0,0 +1,39 @@
+import { h, Component } from 'preact';
+import { bind, inputFieldValueAsNumber } from '../../lib/util';
+import { EncodeOptions } from './encoder';
+
+type Props = {
+  options: EncodeOptions;
+  onChange(newOptions: EncodeOptions): void;
+};
+
+export default class OptiPNGEncoderOptions extends Component<Props, {}> {
+  @bind
+  onChange(event: Event) {
+    const form = (event.currentTarget as HTMLInputElement).closest('form') as HTMLFormElement;
+
+    const options: EncodeOptions = {
+      level: inputFieldValueAsNumber(form.level),
+    };
+    this.props.onChange(options);
+  }
+
+  render({ options }: Props) {
+    return (
+      <form>
+        <label>
+          Level:
```

Comment:
Sorry to bikeshed, but should we go for "Effort" here to match other codecs?

Reply:
Sure